src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTestBase.cs

```diff
@@ -52,7 +52,9 @@ protected static LoopbackServerFactory GetFactoryForVersion(Version useVersion)
             return useVersion.Major switch
             {
 #if NETCOREAPP
+#if HTTP3
                 3 => Http3LoopbackServerFactory.Singleton,
+#endif
```

Comment:
We don't have this #if elsewhere, what is the intent?

Reply:
This shared test base is used in several projects that are unrelated to the http3 work (E.g. WinHttpHandler). It was pulling in the Quic types and now that they're internal I would have had to add all the quic the source files to every test project. I opted to exclude http3 support in unrelated projects.